style.css

```diff
@@ -567,10 +719,52 @@ textarea {
   .primary-text,
   .project-title,
   .project-title-1,
-  .black-line-container {
+  .black-line-container,
+  #mobile-popup {
     display: none;
   }
 
+  /* #mobile-popup {
+    display: none;
+    position: absolute;
+    top: 20px;
+    box-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+    width: 100%;
+    background: #fff;
+    margin: 16px;
+    border: 1px solid #dfe1e6;
+    border-radius: 16px;
+    transform: translate(-73%); */
+
+  /* } */
```

Comment:
- [ ] You have done great job :clap: However, Please do not commit old pieces of code as inline comments. Leaving old pieces of code as comments can clutter the codebase and make it harder to read and understand. Over time, as more code comments accumulate, it becomes challenging to differentiate between actual code and commented-out sections. This clutter can hinder the development process and introduce confusion for developers trying to navigate and maintain the code. :slightly_smiling_face:

Reply:
I have resolved this issue, thank you.